worlds/sc2wol/Regions.py

```diff
@@ -1,8 +1,8 @@
-from typing import List, Set, Dict, Tuple, Optional, Callable, NamedTuple
+from typing import List, Set, Dict, Tuple, Optional, Callable
 from BaseClasses import MultiWorld, Region, Entrance, Location, RegionType
 from .Locations import LocationData
 from .Options import get_option_value
-from worlds.sc2wol.MissionTables import MissionInfo, vanilla_shuffle_order, vanilla_mission_req_table, \
+from .MissionTables import MissionInfo, vanilla_shuffle_order, vanilla_mission_req_table, \
```

Comment:
and maybe also add a note to world api.md that you are supposed to do relative imports *within* the world